terraform/deployments/rds/variables.tf

```diff
@@ -72,3 +72,8 @@ variable "terraform_delete_rds_timeout" {
   description = "Set the timeout time for AWS RDS deletion."
   default     = "2h"
 }
+
+variable "zendesk_2nd_line_email_address" {
```

Comment:
As we're retiring 2ndline - think these emails should just come to platform support?

Reply:
I'm not sure we have "platform support" specific email address. 
There’s `govuk-platform-engineering@digital.cabinet-office.gov.uk`, which is already [present in the main repo](https://github.com/search?q=repo%3Aalphagov%2Fgovuk-infrastructure%20govuk-platform-engineering%40digital.cabinet-office.gov.uk&type=code) _(conscious that if we use that, we won't need the separate private repo shebang)_.